content/agent/proxy.md

```diff
@@ -70,21 +70,21 @@ proxy:
 
 #### Proxy with environment variables
 
-Starting with Agent v6.4, you can set you proxy settings through environment variables:
+Starting with Agent v6.4, you can set your proxy settings through environment variables:
 
-- `DD_PROXY_HTTPS`: Sets a proxy server for `https` queries.
-- `DD_PROXY_HTTP`: Sets a proxy server for `http` queries.
-- `DD_NO_PROXY`: Sets a list of host that should bypass the proxy (a space-separated list).
+* `DD_PROXY_HTTPS`: Sets a proxy server for `https` queries.
+* `DD_PROXY_HTTP`: Sets a proxy server for `http` queries.
```

Comment:
is queries the right word here? should it be traffic?

Reply:
Since it only applies to HTTP traffic, I feel `queries` is better. no ?